packages/cli/templates/react/package.json

```diff
@@ -22,7 +22,7 @@
     "@vitejs/plugin-react": "^3.1.0",
     "eslint": "latest",
     "eslint-plugin-react": "7.30.0",
-    "typescript": "^4.9.3",
+    "typescript": "5.1.6",
```

Comment:
We need to upgrade the typescript inside our template for a reason. It will be explained in the tsconfig.json file.